automat/_methodical.py

```diff
@@ -75,14 +75,27 @@ def _transitionerFromInstance(oself, symbol, automaton):
         setattr(oself, symbol, transitioner)
     return transitioner
 
+def assert_no_code(inst, attribute, f):
+    # the function body must be empty, i.e. "pass" or "return None", which
+    # both yield the same bytecode: LOAD_CONST (None), RETURN_VALUE
+    none_const = f.__code__.co_consts.index(None)
+    # this depends upon the function using None as one of the first 256
+    # constants
+    bytecode = f.__code__.co_code
+    c = [ord(bytecode[i:i+1]) for i in range(len(bytecode))]
+    if c != [100, # LOAD_CONST
+             none_const, 0, # 16-bit index into co_consts
+             83, # RETURN_VALUE
+             ]:
+        raise ValueError("function body must be empty")
```

Comment:
This makes buildbot 3.6 unit test fail.
debugging shows that at that point, c == [100, 1, 83, 0] on py3.6